lib/util/Timer.js

```diff
@@ -1,10 +1,15 @@
 var _DEBUG = typeof DEBUG === 'undefined' ? false : DEBUG;
 var _ASSERT = typeof ASSERT === 'undefined' ? false : ASSERT;
 
-module.exports.now = function() {
+export const now = function() {
```

Comment:
`export function now() { //...` would seem a bit more natural to me (same with `diff` and the functions in common.js